.travis.yml

```diff
@@ -1,16 +1,21 @@
 sudo: false
+
 language: go
+
 go:
-- '1.10.x'
+- '1.12.x'
 
 go_import_path: github.com/andrewkroh/gvm
 
+env:
+  - GO111MODULE=on
+
 install:
-- go get github.com/mitchellh/gox
+- GO111MODULE=off go get github.com/mitchellh/gox
```

Comment:
Is there a way to use go modules with this too?  Having a content checksum on this would be great.

Reply:
I think it would be possible to do a `GO111MODULE=on go get github.com/mitchellh/gox@v1.0.1` and have it added to the go.mod file, but I'm leaning toward not having that listed in go.mod since it's not a dependency of the gvm code.

I haven't done enough with go modules to know if this is a wise decision or not.